pkg/gitparse/gitparse.go

```diff
@@ -126,9 +126,13 @@ func (c *Parser) RepoPath(ctx context.Context, source string, head string, abbre
 	} else {
 		args = append(args, "--all")
 	}
+	if len(excludedGlobs) != 0 {
```

Comment:
Nit: This length check if unnecessary, the inner iteration over the slice will do the same thing.

Reply:
whoops, that's a relic. Good catch